include/gui/gui_overrides_menu.hpp

```diff
@@ -4,25 +4,11 @@
 #include "override_key.hpp"
 
 enum class OverrideButtonType {
-  Album,
   Any_Title,
   Custom_Title,
   AddNew,
```

Comment:
Should probably be Add_New or make everything UPPER_SNAKE_CASE like all other enums

Reply:
Resolved in 531c030